test/es6/exportmodule.js

```diff
@@ -0,0 +1,2 @@
+export var x;

```

Comment:
Oh phew, it was fixed but this comment thread stayed. No problems here. Move along :P

---
In reply to: [70149271](https://github.com/Microsoft/ChakraCore/pull/1254#discussion_r70149271) [](ancestors = 70149271)